### PR TITLE
add aws api gateway integration doc

### PR DESCRIPTION
**Pull request by RazcoDev**

### CODE REVIEW

File: docs/integrations/gateways/aws-api-gateway.mdx [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+sidebar_position: 2
+title: AWS API Gateway x Permit
+---
+
+This document describes how to create an API Gateway that uses a Permit authorizer to authenticate requests.
```
Comment (gemanor) [inline docs/integrations/gateways/aws-api-gateway.mdx:6]:
```suggestion
This document describes how to create an API Gateway that uses a Permit authorizer to authorize requests.
```

File: docs/integrations/gateways/aws-api-gateway.mdx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+## Create a Permit Authorizer
+In order to create an Authorizer that uses Permit, you will need to create a Lambda function that uses Permit to authenticate requests.
```
Comment (gemanor) [inline docs/integrations/gateways/aws-api-gateway.mdx:9]:
```suggestion
In order to create an Authorizer that uses Permit, you will need to create a Lambda function that uses Permit to authorize requests.
```

File: docs/integrations/gateways/aws-api-gateway.mdx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+### Create a Lambda Function
+First, create a new Lambda function - we use Python 3.12 in this example but you can use any runtime that Permit supports with one
+of our [SDKs](/sdk/sdks-overview)
+
+![Lambda creation](/img/integrations/aws/aws-1.png)_
+
+### Create a Layer
```
Comment (gemanor) [inline docs/integrations/gateways/aws-api-gateway.mdx:17]: Add a small description that describe this is required to be in local environment and that you need venv and python installed.

File: docs/integrations/gateways/aws-api-gateway.mdx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    }
+    return asyncio.get_event_loop().run_until_complete(
+        permit.check(user["id"], event["action"], event["resource"])
+        )
+```
+You can see in this example few important things:
+- Make sure to replace `permit_key_xxxxx` with your actual API key.
+- We use the Permit library to check the user's permissions, and specifically the `permit.Check()` method.
+- We receive the user's ID, email, action, and resource from the event, and use them to check the user's permissions.
+- We are using the [cloud PDP](concepts/pdp/#hostedmanaged-cloud-pdp-option) which is hosted by Permit, but keep in mind that it does not support ABAC and RBAC policies. In production, you should [deploy your own PDP](how-to/deploy/deploy-to-production).
```
Comment (gemanor) [inline docs/integrations/gateways/aws-api-gateway.mdx:75]: Move to admonition

File: docs/integrations/gateways/aws-api-gateway.mdx [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+Choose your desired API type, and click "Create API" - we will use the HTTP API in this example.
+
+### Create a New Authorizer
+In the API Gateway console, click on "Authorizers" in the left-hand menu, and then click "Create Authorizer".
+Choose the Lambda function you created earlier, and choose the event payload version you want to use - we will use the Request - header version in this example.
+![Create Authorizer](/img/integrations/aws/aws-5.png)
+
+### Attach the Authorizer to a Resource
+Now, you can attach the authorizer to a resource in your API. Click on "Resources" in the left-hand menu, and then click on the resource you want to attach the authorizer to.
+Then, create a new method for the resource, and choose the authorizer you created earlier.
```
Comment (gemanor) [inline docs/integrations/gateways/aws-api-gateway.mdx:97]: Is it possible to create multiple authorizers there? if so, it's important to mention.

File: docs/integrations/gateways/aws-api-gateway.mdx [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+![Attach Authorizer](/img/integrations/aws/aws-6.png)
+After you have created the method, you can deploy the API by clicking "Deploy" in the top right corner of the API Gateway console.
+
+### Test the API
+Now, you can test the API by sending a request to the endpoint you created. You can use the "Test" tab in the API Gateway console to send a request to the endpoint, and see the response from the Lambda function.
+
```
Comment (gemanor) [inline docs/integrations/gateways/aws-api-gateway.mdx:103]: I would add here an `Advanced` section to explain the principles developers can extend from the tutorial.
For example:
* Tranform better check
* Use a different authorizer for ReBAC/ABAC or Get/other CRUD 
* Deploy the PDP closer to the gateway lamabda

File: docs/integrations/gateways/aws-api-gateway.mdx [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+sidebar_position: 2
+title: AWS API Gateway x Permit
+---
+
+This document describes how to create an API Gateway that uses a Permit authorizer to authenticate requests.
+
+## Create a Permit Authorizer
+In order to create an Authorizer that uses Permit, you will need to create a Lambda function that uses Permit to authenticate requests.
+
+### Create a Lambda Function
```
Comment (gemanor) [inline docs/integrations/gateways/aws-api-gateway.mdx:11]: Add admonition that states users can add a lambda in any language supported by Permit SDKs

**Review by gemanor — CHANGES_REQUESTED** [review]
Good job, left some comments

File: docs/integrations/gateways/aws-api-gateway.mdx (hunk 5 above)
Comment (RazcoDev) [reply docs/integrations/gateways/aws-api-gateway.mdx:97]: It is not :)

File: docs/integrations/gateways/aws-api-gateway.mdx (hunk 6 above)
Comment (RazcoDev) [reply docs/integrations/gateways/aws-api-gateway.mdx:103]: done

File: docs/integrations/gateways/aws-api-gateway.mdx (hunk 1 above)
Comment (RazcoDev) [reply docs/integrations/gateways/aws-api-gateway.mdx:6]: done

File: docs/integrations/gateways/aws-api-gateway.mdx (hunk 2 above)
Comment (RazcoDev) [reply docs/integrations/gateways/aws-api-gateway.mdx:9]: done

File: docs/integrations/gateways/aws-api-gateway.mdx (hunk 7 above)
Comment (RazcoDev) [reply docs/integrations/gateways/aws-api-gateway.mdx:11]: done

File: docs/integrations/gateways/aws-api-gateway.mdx (hunk 3 above)
Comment (RazcoDev) [reply docs/integrations/gateways/aws-api-gateway.mdx:17]: done

File: docs/integrations/gateways/aws-api-gateway.mdx (hunk 4 above)
Comment (RazcoDev) [reply docs/integrations/gateways/aws-api-gateway.mdx:75]: done

**Review by gemanor — APPROVED** [review]

**Pull request merged**